### ISSUE TITLE
Error during pip installation

### ISSUE THREAD

**Issue by ValeryPolyakov** [ccsb-scripps/AutoDock-Vina#194]
This happens on both Windows and Linux:

Collecting vina
  Using cached vina-1.2.3.tar.gz (95 kB)
  Installing build dependencies ... done
  Getting requirements to build wheel ... error
  error: subprocess-exited-with-error

  × Getting requirements to build wheel did not run successfully.
  │ exit code: 1
  ╰─> [3 lines of output]
      fatal: not a git repository (or any of the parent directories): .git
      Version found 1.2.3 (from __init__.py)
      error in vina setup command: 'python_requires' must be a string containing valid version specifiers; Invalid specifier: '>=3.5.*'
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: subprocess-exited-with-error

× Getting requirements to build wheel did not run successfully.
│ exit code: 1
╰─> See above for output.

note: This error originates from a subprocess, and is likely not a problem with pip.

**Comment by Kresten1** [comment]
To fix the issue download vina.1.2.3.tar.gz from pypi.org. Then run "tar -xf vina-1.2.3.tar.gz" in the directory with the file. A folder named "vina-1.2.3" should appear.  Edit the setup.py file by removing the .* in line 361. 
Then change directory into the vina folder and run "python setup.py install" 

Hope that helps

**Comment by Kobra173** [comment]
tengo el mismo error

**Comment by Kresten1** [comment]
> 

In the Vina-1.2.3 folder, a file named "Setup.py" should be present. Open that file with a code editor, such as Notepad or the likes. Go down to line 361, is should say something like" python=<3.5.* " delete the .* And save the file. Then run the setup.py file trough the terminal

**Comment by tonyliang19** [comment]
> > Para solucionar el problema, descargue vina.1.2.3.tar.gz desde pypi.org. Luego ejecute "tar -xf vina-1.2.3.tar.gz" en el directorio con el archivo. Debería aparecer una carpeta llamada "vina-1.2.3". Edite el archivo setup.py eliminando .* en la línea 361. Luego cambie el directorio a la carpeta vina y ejecute "python setup.py install"
> > Espero que ayude
> 
> podrias esplicarme que es exactamente lo que debo editar?

Lo que se está haciendo es remover el punto asterico o `.*` de la linea 361 del `vina-1.2.3/setup.py`.

**Comment by tonyliang19** [comment]
> To fix the issue download vina.1.2.3.tar.gz from pypi.org. Then run "tar -xf vina-1.2.3.tar.gz" in the directory with the file. A folder named "vina-1.2.3" should appear. Edit the setup.py file by removing the .* in line 361. Then change directory into the vina folder and run "python setup.py install"
> 
> Hope that helps

That did not work for me in `python3.11`, what version worked for you? I looked at `setup.py`, and only saw Python 3.5 - Python 3.9?

**Comment by tonyliang19** [comment]
With `python==3.9.7`, `pip install vina` worked for me. If you are using conda, could create it by the following:

```bash
conda create -y -n vina python=3.9.7
conda install -y -c conda-forge numpy swig boost-cpp sphinx sphinx_rtd_theme
pip install vina
```

**Comment by diogomart** [comment]
Fixed in #167
